docs/developer-docs/integrations/vetkeys/using-vetkeys.md

```diff
@@ -22,30 +22,42 @@ git clone https://github.com/dfinity/examples/
 Then navigate into the directory specifically for this project:
 
 ```
-cd examples/rust/vetkd
+cd examples/motoko/vetkd
```

Comment:
both folders exist and contain code:

https://github.com/dfinity/examples/tree/master/rust/vetkd
https://github.com/dfinity/examples/tree/master/motoko/vetkd

the reader change choose which one to run